machine-learning/README.md

```diff
@@ -1,4 +1,100 @@
-# Machine Learning (PyTorch, Tensorflow)
-all the tools and guidance needed to get up and running at a hackathon!
+# Machine Learning (Keras, PyTorch, Tensorflow)
 
+## What is Machine Learning and Why Would I Use it?
+Machine Learning is a broad umbrella for methods of developing artificial intelligence, which is generally used to find patterns in large data sets based on mathematical and statistical calculations. This becomes very useful when a phenomena cannot be modeled by a deterministic algorithm.   
```

Comment:
There're a lot of big words here. Can you reword it so that someone with no ML experience will understand it? Like when I read this I'm just like ehhh that kinda made sense but like I don't wanna sit here understanding it.